docs/src/integration-keda.md

```
@@ -0,0 +1,41 @@
+- Make sure you have gone through the [getting started](getting-started.md) guide. We will extend the same setup for this integration.
+- KEDA installed in the cluster - [KEDA Installation](https://keda.sh/docs/latest/deploy/)
+
+## Steps
+
+### 1. Create a keda scaler for the service
+
+Let's create a keda scaler for the httpbin service.
+
+``` shell
+kubectl apply -f ./playground/config/demo-application-keda.yaml
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)